src/MissionManager/SurveyMissionItem.cc

```diff
@@ -1062,7 +1062,7 @@ bool SurveyMissionItem::_appendMissionItemsWorker(QList<MissionItem*>& items, QO
         MissionItem* item = new MissionItem(seqNum++,
```

Comment:
My point is, can we remove the `_imagesEverywhere` check so that the camera actually turns off at the end of any survey?